frontend/src/Navigation/index.jsx

```diff
@@ -85,11 +85,18 @@ const AuthLoadingStack = createStackNavigator({
   },
 }, hideHeaderOptions);
 
+const MapStack = createStackNavigator({
+  Map: {
+    screen: Map,
+    path: 'map/',
+  },
+}, hideHeaderOptions);
+
```

Comment:
Why map has its own stack? What else will be in this stackNavigator?

Reply:
I don't know. Is there any other solution?
I use this stack navigator for further extension.